pennylane/_qubit_device.py

```diff
@@ -280,6 +280,17 @@ def statistics(self, observables):
             elif obs.return_type is Probability:
                 results.append(self.probability(wires=obs.wires))
 
+            elif obs.return_type is State:
+                try:
+                    state = self.state
+                except NotImplementedError:
+                    state = None
+
+                if state is None:
+                    raise ValueError("The state is not available in the current device")
```

Comment:
I was going to suggest combining it into a single
```python
except NotImplementedError:
    raise ValueError("The state is not available in the current device")
```

But I suppose this is better, since some devices might return `state=None`?

Reply:
Yes, I originally thought that but found out the ones that return `None`